python/dgl/distributed/optim/pytorch/sparse_optim.py

```diff
@@ -32,7 +32,8 @@ def __init__(self, params, lr):
             self._rank = th.distributed.get_rank()
             self._world_size = th.distributed.get_world_size()
         else:
-            assert 'th.distributed shoud be initialized'
+            self._rank = 0
```

Comment:
This is for standalone single trainer?

Reply:
this is required at least for test code where we cannot initialize pytorch distributed.